__tests__/app.test.js

```diff
@@ -73,6 +73,93 @@ describe("App", () => {
           });
         });
     });
+    // Query tests
+    test("200: Responds with the array of review objects, all of which should have all the category types", () => {
+      return request(app)
+        .get("/api/reviews?category=dexterity")
```

Comment:
I see what you're doing here, but based on the description - shouldn't it now just be `/api/reviews`?